desktop/assets/texts/fr-FR/settings.ftl

```diff
@@ -70,7 +70,7 @@ scale-mode-force = Forcer
 scale-mode-force-tooltip = Empêche le film de changer de mode de mise à l'échelle, en le verrouillant sur le paramètre sélectionné.
 player-version = Version du lecteur
 player-runtime = Type de lecteur
-player-runtime-flash = Flash Player
+player-runtime-flash = Lecteur Flash
```

Comment:
Should this be translated? Adobe doesn't translate runtime names in their docs. I see that it's translated in some languages, in some it isn't

Reply:
Good catch.☺️I did almost all of the French translation, and I agree that this shouldn't be translated in this context.

I'm also unsure about the change related to the letterbox setting: while the translation itself is correct, the word "letterbox" is usually not translated (eg. https://fr.wikipedia.org/wiki/Letterbox).

Reply:
Reverted this, also updated translations for other languages where this shouldn't be translated